nemo/lightning/pytorch/strategies.py

```diff
@@ -491,6 +505,22 @@ def training_step(self, dataloader_iter, *args: Any, **kwargs: Any) -> STEP_OUTP
 
             return out
 
+    @override
+    def optimizer_step(
+        self,
+        optimizer: torch.optim.Optimizer,
+        closure: Callable[[], Any],
+        model: Optional[Union["pl.LightningModule", nn.Module]] = None,
+        **kwargs: Any,
+    ) -> Any:
+        optimizer_output = super().optimizer_step(optimizer, closure, model, **kwargs)
+
+        if isinstance(optimizer, McoreDistributedOptimizer):
+            _, grad_norm = optimizer_output
+            self.lightning_module.log('grad_norm', grad_norm, batch_size=1)
+
+        return optimizer_output
```

Comment:
`return optimizer_output` or `return optimizer_output[0]` ?

Reply:
Yeah, this was my question regarding if it makes any difference. not sure if PTL cares about the output of this. I can make the change to be safe.